Fixed the image quality in DINE-IN page.

ApoorvaSunkad opened the pull request:
# 🚀 Pull Request

## Description

Hi, 

The image quality of 2 images on the DINE-IN page (see the screenshots) was poor. It did not match other image quality to enhance the overall image quality and the website. I have added 2 new images of a similar type to the previous image in the images folder and then added them to dine-in.html also to fit the aspect ratio of the container I have provided the property "object-fit: cover".

I have also added comments for better understanding.

Kindly merge my Pull Request.

- Fixes #701 
- Closes #701 


## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

## Screenshot of final output/video

BEFORE
![icy1](https://github.com/user-attachments/assets/c3651dc3-009e-4e21-be4f-2b0b1424da6a)




AFTER
![icy2](https://github.com/user-attachments/assets/a472788b-375b-4e31-b0d9-0c2f6df0d392)
![icy3](https://github.com/user-attachments/assets/717c4657-6910-4e87-a475-d403a9878a7d)

Thank you for allowing me to contribute and learn new things. Happy Coding!

ApoorvaSunkad commented:
Hi,

Resolved the conflict occurred. Kindly check once and merge the Pull Request.

Thank you!

DharshiBalasubramaniyam reviewed (changes requested):
Why you  updated the styles.css?

ApoorvaSunkad commented:
Hi @DharshiBalasubramaniyam ,

When I was searching for the dine-in.css style page I could not find it, hence I updated it in styles.css. Upon checking today, I found the dine-in.css page and will update it, and remove the changes from styles.css. I apologize for my mistake this is my 1st open-source contribution at GSSOC'24. 

Can I update the dine-in.css page with my changes? Could you please let me know?

Thank You!

ApoorvaSunkad commented:
Hi @DharshiBalasubramaniyam ,

I removed the style applied in the styles.css page and added it in dine-in.css. Kindly let me know if this is correct.

Thank you!

ApoorvaSunkad commented:
Hi @DharshiBalasubramaniyam ,

Thank you for allowing me to contribute to the project. I enjoyed working on it.

Happy Coding!
